tasks/bump.js

```diff
@@ -105,10 +105,19 @@ module.exports = function(grunt) {
         var content = grunt.file.read(file).replace(
           VERSION_REGEXP,
           function(match, prefix, parsedVersion, namedPre, noNamePre, suffix) {
+            var metadata = "";
             var type = versionType === 'git' ? 'prerelease' : versionType;
+            if(opts.metadata) {
+              if(/^([0-9a-zA-Z-]+\.{0,1})*$/.test(opts.metadata)) {
```

Comment:
you can do this after the version line. I wanted to say use semver.valid() but that won't tell you if the metadata isnt valid so this is ok